src/Symfony/Component/CssSelector/XPath/Extension/ExtensionInterface.php

```diff
@@ -60,6 +60,13 @@ public function getPseudoClassTranslators(): array;
      */
     public function getAttributeMatchingTranslators(): array;
 
+    /**
+     * Returns relative combinators translators.
+     *
+     * @return callable[]
```

Comment:
I suggest using `array<string, callable>` to describe the type of keys. Or even better, `array<string, callable(XPathExpr, XPathExpr): XPathExpr>` to describe the callable signature as well.